migrations/versions/06c32cf470e7_.py

```diff
@@ -0,0 +1,154 @@
+"""empty message
```

Comment:
Can you check what is the policy wrt to keeping such files in the repo ? I believe we should commit them, but better to check.

Reply:
It's clearily said on flask-migrate site to keep them in version control.